examples/api-samples/src/browser/output/sample-output-channel-with-severity.ts

```diff
@@ -16,26 +16,83 @@
 import { FrontendApplicationContribution } from '@theia/core/lib/browser';
 import { inject, injectable, interfaces } from 'inversify';
 import { OutputChannelManager, OutputChannelSeverity } from '@theia/output/lib/common/output-channel';
+import { CommandContribution, CommandRegistry } from '@theia/core/lib/common/command';
```

Comment:
the name of this file should be changed now since it does not only test the severities, right?

Reply:
Good point, but the corresponding commit will be dropped from the PR.